src/middlewared/middlewared/plugins/system.py

```diff
@@ -1444,6 +1455,19 @@ async def _event_system(middleware, event_type, args):
         # If it is not defined yet, it will try to define
         if birthday is None:
             asyncio.ensure_future(_update_birthday(middleware))
+
+        if osc.IS_LINUX:
+            if (await middleware.call('system.advanced.config'))['kdump_enabled']:
+                cp = await run(['kdump-config', 'status'], check=False)
+                if cp.returncode:
+                    middleware.logger.error('Failed to retrieve kdump-config status: %s', cp.stderr.decode())
```

Comment:
Again, alert won't be deleted in this code path

Reply:
There is a possibility it is configured in this case. We were just unable to retrieve the status of kdump-config which in various scenarios i have tested should not happen and is just a fail safe